### DISCUSSION TITLE
Add a way to configure community containers

### DISCUSSION THREAD

**Discussion by AiroPi** [nextcloud/all-in-one#4280] [Ideas] [answered]
Community containers are a very cool feature, but I was wondering if we could add some configurations for them. For example, add additional volumes, or set some environment variables ?
Maybe this is already planned ? I know that community containers are still in beta.

**Comment by szaimen** [comment, marked as answer]
> Community containers are a very cool feature,

Thanks!
> but I was wondering if we could add some configurations for them. For example, add additional volumes, or set some environment variables ?

This is tracked in https://github.com/nextcloud/all-in-one/discussions/3889. Help is appreciated!
